docs/src/ref_impl/build_and_run.md

```diff
@@ -0,0 +1,104 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Build and Run
+
+The canonical URL for the upstream Ghaf git repository is <https://github.com/tiiuae/ghaf>. To try Ghaf, you can build it from the source.
+
+Our supported targets are the following:
+
+| Hardware         | Architecture     | Scope         |
+|---               |---               |---            |
+| Generic Intel    | x86              | QEMU (VM)     |
```

Comment:
We have x86 hardware support nowadays as well. The NUC target is generic in practice and it's been lately installed successfully also to laptops.